### PR TITLE
Anchor: Update branding in the My Home podcasting card.

### PR DESCRIPTION
**Pull request by kwight**
Anchor recently updated their branding. This PR updates their logo in the podcasting My Home card to be compliant with their new usage guidelines.

Screen Shot 2021-02-04 at 2 41 45 PM

Fixes 441-gh-Automattic/dotcom-manage

**Testing Instructions**
* Load this branch in a local dev environment, or use the calypso.live branch with `?flags=anchor-fm-dev`.
* One a site with at least three published posts, cycle through and dismiss all of the My Home cards until you get to the podcasting card.
* Verify the Anchor logo is the updated purple one shown above.

### CODE REVIEW

File: client/assets/images/customer-home/illustration--task-podcasting.svg [hunk 1]
```diff
@@ -1,81 +1,80 @@
-<svg width="257" height="157" viewBox="0 0 257 157" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (kwight) [inline client/assets/images/customer-home/illustration--task-podcasting.svg:1]: @sfougnier are these sizes differences (and the ones at the bottom) a problem?.. I can never tell (it looks great, so...). ¯\_(ツ)_/¯

**Comment by ghost** [comment]
I'm having a hard time getting the card to trigger. I'm on a launched site with 3 published posts, but still not able to get the Anchor card to appear on my home. Is this the Calypso live branch I should be using? https://calypso.live/?branch=update/anchor-branding

**Comment by kwight** [comment]
> I'm having a hard time getting the card to trigger

@sfougnier If you use the calypso.live branch, you need to add the Anchor dev flag (which is only on by default in Calypso local dev): https://hash-2333e1ec466fbce309be968d50795ba6645bf6d7.calypso.live/?flags=anchor-fm-dev

**Comment by sixhours** [comment]
Like Saul, I can't get this to load on calypso.live, only my local dev environment:

Screen Shot 2021-02-02 at 5 20 55 PM

I even added the flag to my site through the blog rc, but that didn't help...

**Comment by kwight** [comment]
Erm, OK – I'm not seeing it work either, and I don't see an immediate reason (are `flags` params not working?). Well, it's all to duplicate the local env, so if you see it as expected on local @sixhours , we should be good.

**Comment by blackjackkent** [comment]
I'm also having trouble getting the card to show up even in my local environment, for some reason. (Even with the flag in the URL.) However, the code diff looks fine to me so if things are working for Caroline, it seems fine to me. :)

**Comment by sixhours** [comment]
> Well, it's all to duplicate the local env, so if you see it as expected on local @sixhours , we should be good.

True, hard for other designers without local environments to see changes, though. :-/ I'm happy to act as a go-between or help folks get testing environments set up if it helps Anchor get to the finish line, just let me know @sfougnier.

**Comment by ghost** [comment]
🎉 I was able to view this. Thanks @sixhours!

Overall it looks ok - I did notice the bottom horizontal line is a bit off compared to the original illustration and was able to see what in the SVG may be causing it. New file below:
[podcast.svg.zip](https://github.com/Automattic/wp-calypso/files/5920045/podcast.svg.zip)

The way things are scaling on mobile is a bit strange. Can we hide the illustration on smaller screens?
Screen Shot 2021-02-03 at 11 41 28 AM

**Comment by kwight** [comment]
> The way things are scaling on mobile is a bit strange. Can we hide the illustration on smaller screens?

@sfougnier We already do – I'm guessing you did a manual window resize? I can't remember why it works this way, but if the page is loaded at less than ~1055px, we drop the image; and mobile views get a different layout that only prompts for installing the app.

| 1055px | 1058px | Mobile |
| --- | --- | --- |
| Screen Shot 2021-02-03 at 10 54 43 AM | Screen Shot 2021-02-03 at 10 54 08 AM | Screen Shot 2021-02-03 at 10 45 48 AM

**Comment by ghost** [comment]
Oh, cool. I was resizing it in Inspect mode. Looks better in the
screenshots you’re sharing 🙂

**Comment by sixhours** [comment]
IIRC that happens because we determine whether to show the image with JS rather than CSS, so a window resize doesn't have an effect.

**Comment by kwight** [comment]
@sfougnier @sixhours Sweet – if we're good, can one of you approve please?

**Comment by ghost** [comment]
Just one more thing: we need to update that SVG to reflect the new HEX colors for the Anchor branding.
[podcast.svg.zip](https://github.com/Automattic/wp-calypso/files/5928777/podcast.svg.zip)

**Comment by kwight** [comment]
> we need to update that SVG to reflect the new HEX colors for the Anchor branding.

Sweet, done!

Screen Shot 2021-02-04 at 2 41 45 PM

**Comment by kwight** [comment]
Merging this so it's ready for testing – I can follow up if there's anything still to do.

**Pull request merged**